impress-org/givewp-facebook-pixel-tracking
# Facebook Pixel Tracking add-on now uses the add-on boilerplate

Resolves #7 

## Description
This PR refactors work already done with the FBPT add-on so that it uses the GiveWP add-on boilerplate.

## Affects
The PR affects virtually all aspects of the plugin, since it is an entire refactoring into a different boilerplate/starting point.

## Pre-review Checklist

## Testing Instructions
1. Install/activate the Facebook Pixel Tracking addon.
2. Create a multi-step form, and a legacy form.
3. With the console open, complete a donation in either form.
4. Check that the message "FBPT script loaded!" is logged to the console only when the receipt is shown.
